ahcer/src/app/create-episode/create-episode.component.ts

```diff
@@ -57,10 +67,12 @@ export class CreateEpisodeComponent implements OnInit{
       controls[label+' Checkbox'] = false;
       if (label!="Seizure" && label!="Loss of Consciousness")
         controls[label+' Dropdown'] = ['']
+      controls[label+' Time'] = null
     }
     let options = {
```

Comment:
This code, lines 72-103 is identical to 57-88. Could we abstract this out into a method/property and not duplicate?

Reply:
57-88 in edit-episode. Wonder if you could create an episode-helper.ts and move these lines to that? Looks like there are several other methods that could end up there as well?

Reply:
also wanted to address this since I had to change the code in two different places for so many times already, and there are many duplicates between create and update for each of the four models. I think it's better if we move all CRUD components (maybe except for user) to folders with their model name, create helpers in each folder, find duplicate codes and move them to helpers.  This would make the project more organized as the number of components increases.